source/puppet/4.4/reference/system_requirements.markdown

```diff
@@ -4,87 +4,78 @@ title: "Puppet 4.4 System Requirements"
 
-{% include pup43_platforms_fedora.markdown %}
+All Puppet masters must be available to agents over the network. See the [pre-installation instructions][preinstall-network] for details about the required DNS and port configuration for masters.
 
-### Windows
+The Puppet master service built into the `puppet-agent` package is a resource-intensive service that performs best on a robust dedicated server. To manage more than 10 nodes, the master server should have at least two processor cores and 1 GB of RAM. To manage more than 1,000 nodes, the server should have at least 4 processor cores and 4 GB of RAM.
 
-{% include pup43_platforms_windows.markdown %}
+To scale Puppet beyond more than a few hundred nodes, however, we recommend using [Puppet Server]({{puppetserver}}) instead of the built-in Puppet master. Puppet Server is a much more efficient implementation of the Puppet master service. See [its documentation]({{puppetserver}}/install_from_packages.html#system-requirements) for system requirements and installation instructions.
 
-### Mac OS X
+{% include agent_lifecycle.md %}
 
-{% include pup43_platforms_osx.markdown %}
+## Platforms without packages
 
-## Platforms Without Packages
+Puppet and its prerequisites can also run on the following platforms, but we do not provide official open-source packages or perform automated testing of open source Puppet for them.
 
-Puppet and its prerequisites are known to run on the following platforms, but we do not provide official open source packages or perform automated testing. For platforms supported in Puppet Enterprise, see its [System Requirements]({{pe}}/install_system_requirements.html#supported-operating-systems).
+### Linux
 
-### Other Linux
+-   SUSE Linux Enterprise Server, version 11 and higher
+-   Gentoo Linux
+-   Mandriva Corporate Server 4
+-   Arch Linux
 
-* SUSE Linux Enterprise Server, version 11 and higher
-* Gentoo Linux
-* Mandriva Corporate Server 4
-* Arch Linux
+### Unix
 
-### Other Unix
+-   Oracle Solaris, version 10 and higher (Puppet performs limited automated testing on Solaris 11)
+-   AIX, version 5.3 and higher
+-   FreeBSD, version 4.7 and later
+-   OpenBSD, version 4.1 and later
+-   HP-UX
 
-* Oracle Solaris, version 10 and higher (Puppet performs limited automated testing on Solaris 11)
-* AIX, version 5.3 and higher
-* FreeBSD 4.7 and later
-* OpenBSD 4.1 and later
-* HP-UX
+> **Note:** For platforms supported in Puppet Enterprise, see its [System Requirements][pe-requirements].
 
-## Basic Requirements
+Puppet also makes the `puppet-agent` packaging process available as an open source project. For information about building the package yourself, see the [`puppetlabs/puppet-agent` repository on GitHub](https://github.com/puppetlabs/puppet-agent).
```

Comment:
Confirm that we can recommend the `puppet-agent` repo to open source users who want to build from source, as there isn't a guide to build Puppet 4 from source.